styles/variables-gallium.less

```
@@ -136,11 +139,17 @@
 @agate-mediacontrols-icon-padding: 0 30px 0 30px;
 @agate-mediacontrols-tiny-icon-font-size: (@agate-mediacontrols-tiny-icon-height * 2);
 @agate-mediacontrols-playpause-button-bg-border-radius: 90px;
+@agate-mediacontrols-playpause-button-bg-height: 90px;
+@agate-mediacontrols-playpause-button-bg-width: 90px;
+@agate-mediacontrols-playpause-button-client-width: 90px;
 @agate-mediacontrols-playpause-button-height: 90px;
 @agate-mediacontrols-playpause-button-margin: 0 30px;
+@agate-mediacontrols-playpause-button-padding: 0;
+@agate-mediaplayer-height: 132px;
 @agate-mediaplayer-times-font-size: 27px;
 @agate-mediaplayer-times-line-height: 36px;
 @agate-mediaplayer-times-margin: 0 12px 0 12px;
+@agate-mediaplayer-times-padding-top: 0px;
```

```suggestion
@agate-mediaplayer-times-padding-top: 0;
```
Please remove 'px' for 0